packages/blade/src/components/Box/__tests__/Box.ssr.test.tsx

```diff
@@ -20,12 +22,17 @@ describe('<Box />', () => {
           class="BaseBoxweb__BaseBox-sc-1icfu8j-0 cVhZTX"
           data-blade-component="Box"
           display="flex"
-        />
+        >
+          children test!
+        </div>
       </div>
     `);
   });
 
   it('should throw error for unsupport values', () => {
+    // Ignoring error from console to not show it while running tests
```

Comment:
Can you add do this instead:
beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
afterAll(() => jest.restoreAllMocks());

Reply:
https://github.com/razorpay/blade/pull/1016#discussion_r1122624626 avoiding to disable console error for other tests because of this.

Only required in this error scenario test.